business/istio_config.go

```diff
@@ -310,7 +310,7 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 
 	go func() {
 		defer wg.Done()
-		// MeshPolicies are not namespaced. They will be only listed for an Istio namespace.
+		// MeshPeerAuthentications are not namespaced. They will be only listed for an Istio namespace.
 		// Only listed in non Maistra environments.
 		if criteria.IncludeMeshPolicies && config.IsIstioNamespace(criteria.Namespace) && !in.k8s.IsMaistraApi() {
```

Comment:
This is keeping the criteria.IncludeMeshPolicies shouldn't the wording on the criteria be changed ?
Just asking, but perhaps that may bring to have a mix of meshpeerauthentication + meshpolicies in the handlers.

Reply:
Actually, since Policies and MeshPolicies will still be supported (not validated, not used for mtls icon computing) in the Istio Config List and Istio Config editor, I shouldn't have renamed this one. It should be kept as it is.